models/article.models.js

```diff
@@ -0,0 +1,21 @@
+const db = require("../db/connection");
+
+exports.selectArticleById = (article_id) => {
+ if (!article_id.match(/^\d+$/)) {
```

Comment:
This is a cool way to do it with regex! However, it may be worth considering how we can do it using a PSQL error.